pkgs/desktops/deepin/go-package/dde-daemon/default.nix

```
@@ -67,6 +67,9 @@ buildGoPackage rec {
     substituteInPlace inputdevices/layout_list.go \
       --replace "/usr/share/X11/xkb" "${xkeyboard_config}/share/X11/xkb"
 
+    substituteInPlace system/uadp/crypto.go \
+      --replace "/usr/share/uadp" "/run/current-system/sw/share/uadp"
```

```suggestion
      --replace "/usr/share/uadp" "/var/lib/dde-daemon/uadp"
```
The /usr/share hierarchy is for  ***read-only*** architecture independent data files,  dde-daemon does not strictly comply with FHS


https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch04s11.html
https://refspecs.linuxfoundation.org/FHS_3.0/fhs/ch05s08.html